packages/react-renderer-demo/src/doc-components/additional-component-text.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GenericMuiComponent from '../helpers/generic-mui-component';
+
+const Checkbox = ({ activeMapper, component }) => {
+  try {
+    const Text = require(`@docs/doc-components/examples-texts/${activeMapper}/${component}.md`)?.default;
```

Comment:
@rvsia will you run analyze on the docs bundle? I hope I am wrong but this might pull some additional stuff for each page.

Reply:
```
Page                                                           Size     First Load
┌ ○ /                                                          23.3 kB      285 kB
├   /_app                                                      18.7 kB      241 kB
├ ○ /404                                                       1.17 kB      242 kB
├ ○ /404                                                       1.17 kB      242 kB
├ ○ /components/field-array                                    1.29 kB      282 kB
├ ○ /components/field-provider                                 1.87 kB      271 kB
├ ○ /components/form                                           554 B        270 kB
├ ○ /components/form-spy                                       569 B        270 kB
├ ○ /components/form-template                                  1.7 kB       282 kB
├ ○ /components/renderer                                       1.96 kB      272 kB
├ ○ /development-setup                                         1.51 kB      271 kB
├ ○ /faq                                                       1.55 kB      282 kB
├ ○ /hooks/use-field-api                                       2.17 kB      272 kB
├ ○ /hooks/use-form-api                                        1.36 kB      271 kB
├ ○ /input-meta                                                2.16 kB      243 kB
├ ○ /installation                                              895 B        270 kB
├ ○ /introduction                                              2.05 kB      282 kB
├ ○ /live-editor                                               144 kB       412 kB
├ ○ /mappers/action-mapper                                     1.11 kB      281 kB
├ ○ /mappers/ant-component-mapper                              2.89 kB      272 kB
├ ○ /mappers/blueprint-component-mapper                        3.32 kB      273 kB
├ ○ /mappers/carbon-component-mapper                           3.46 kB      273 kB
├ ○ /mappers/checkbox                                          478 B        277 kB
├ ○ /mappers/checkbox-multiple                                 523 B        277 kB
├ ○ /mappers/component-api                                     4.97 kB      275 kB
├ ○ /mappers/custom-mapper                                     2.94 kB      283 kB
├ ○ /mappers/date-picker                                       520 B        277 kB
├ ○ /mappers/dual-list-select                                  521 B        277 kB
├ ○ /mappers/field-array                                       668 B        277 kB
├ ○ /mappers/file-input                                        1.21 kB      282 kB
├ ○ /mappers/global-component-props                            737 B        281 kB
├ ○ /mappers/mui-component-mapper                              2.77 kB      272 kB
├ ○ /mappers/pf3-component-mapper                              2.79 kB      272 kB
├ ○ /mappers/pf4-component-mapper                              2.62 kB      272 kB
├ ○ /mappers/plain-text                                        488 B        277 kB
├ ○ /mappers/radio                                             505 B        277 kB
├ ○ /mappers/schema-validator-mapper                           1.2 kB       282 kB
├ ○ /mappers/select                                            570 B        277 kB
├ ○ /mappers/slider                                            424 B        277 kB
├ ○ /mappers/sub-form                                          410 B        277 kB
├ ○ /mappers/suir-component-mapper                             3.01 kB      273 kB
├ ○ /mappers/switch                                            471 B        277 kB
├ ○ /mappers/tabs                                              641 B        277 kB
├ ○ /mappers/text-field                                        491 B        277 kB
├ ○ /mappers/textarea                                          482 B        277 kB
├ ○ /mappers/time-picker                                       463 B        277 kB
├ ○ /mappers/validator-mapper                                  1.02 kB      281 kB
├ ○ /mappers/wizard                                            775 B        277 kB
├ ○ /migration-guide                                           2.48 kB      283 kB
├ ○ /optimization                                              1.42 kB      271 kB
├ ○ /releases                                                  6.03 kB      247 kB
├ ○ /schema/actions                                            503 B        270 kB
├ ○ /schema/and                                                879 B        281 kB
├ ○ /schema/async-validator                                    853 B        281 kB
├ ○ /schema/clear-on-unmount                                   1.05 kB      281 kB
├ ○ /schema/cleared-value                                      708 B        281 kB
├ ○ /schema/complex-condition-example                          479 B        281 kB
├ ○ /schema/condition-actions                                  692 B        270 kB
├ ○ /schema/condition-nesting                                  615 B        281 kB
├ ○ /schema/condition-schema                                   918 B        271 kB
├ ○ /schema/condition-sequence                                 885 B        281 kB
├ ○ /schema/condition-set                                      659 B        270 kB
├ ○ /schema/condition-visible                                  571 B        270 kB
├ ○ /schema/constants                                          973 B        271 kB
├ ○ /schema/custom-validator                                   696 B        281 kB
├ ○ /schema/data-types                                         816 B        281 kB
├ ○ /schema/initialize-on-mount                                1.08 kB      281 kB
├ ○ /schema/introduction                                       2.47 kB      272 kB
├ ○ /schema/is                                                 605 B        281 kB
├ ○ /schema/is-empty                                           706 B        281 kB
├ ○ /schema/is-not-empty                                       710 B        281 kB
├ ○ /schema/length-validator                                   476 B        281 kB
├ ○ /schema/not                                                770 B        281 kB
├ ○ /schema/not-match                                          647 B        281 kB
├ ○ /schema/number-value-validator                             483 B        281 kB
├ ○ /schema/or                                                 816 B        281 kB
├ ○ /schema/overwriting-default-message                        571 B        281 kB
├ ○ /schema/pattern                                            713 B        281 kB
├ ○ /schema/pattern-validator                                  474 B        281 kB
├ ○ /schema/record-level-validation                            648 B        281 kB
├ ○ /schema/required-validator                                 476 B        281 kB
├ ○ /schema/resolve-props                                      2.1 kB       282 kB
├ ○ /schema/url-validator                                      472 B        281 kB
├ ○ /schema/validator-mapper                                   503 B        270 kB
├ ○ /testing                                                   3.27 kB      284 kB
└ ○ /typescript                                                996 B        271 kB
+ shared by all                                                241 kB
  ├ static/pages/_app.js                                       18.7 kB
  ├ chunks/02f3b3897fabcb4f16c6eeae13edaf074491334c.d2fe81.js  27.6 kB
  ├ chunks/06166da75c7310003ec9ca2b4e9b444e0d804564.ff5e75.js  4.41 kB
  ├ chunks/09115e561dcf9225013d1122878366ee3c97caaa.f1a77b.js  35.7 kB
  ├ chunks/091f0912a8a524dbbfb317164a6896bda78e423b.5d4996.js  2.83 kB
  ├ chunks/12ba830745348f8c179cf3e3f10caa7c6d239cd1.eb744c.js  2.92 kB
  ├ chunks/251fffa789d834d424164fbad17009beb18150c7.3db009.js  10.3 kB
  ├ chunks/2feafbf00c3bb06c83d7c8c0fad5c25ebca44b99.f1f456.js  5.09 kB
  ├ chunks/3c329624537cde6aa5543b776aa8633eef7962e2.ecf069.js  3.36 kB
  ├ chunks/498c065ec08748b96284c233f1510da422dd252d.cc0df5.js  3.54 kB
  ├ chunks/4f9b5c1d8611fad6daecfc12fc89708468f93dea.939350.js  5.79 kB
  ├ chunks/969568ce9dcc5f74e84f5b4fbeb909968171cccc.3e2b48.js  3.05 kB
  ├ chunks/97dabdf1af30ea90a353ca0f9f5af66be58439fd.a5d7c9.js  2.96 kB
  ├ chunks/c7b526918df76d8fbb50d6a1f88e6226e9e4c79b.1e738e.js  2.62 kB
  ├ chunks/de5fc0a1181c765f2dec423e1158c14b5c9ab080.2c5ed7.js  42 kB
  ├ chunks/f2b7fd54a724a853e247cdcc67f4b547fbf7b878.6aeaf9.js  9.29 kB
  ├ chunks/f9d604c9e6022a73e30b564a1ca7a329b14f6014.30e735.js  10.7 kB
  ├ chunks/framework.a97ce7.js                                 40.9 kB
  ├ runtime/main.621716.js                                     6.06 kB
  └ runtime/webpack.d0d62f.js                                  2.92 kB

λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
○  (Static)  automatically rendered as static HTML (uses no initial props)
●  (SSG)     automatically generated as static HTML + JSON (uses getStaticProps)
```